store/link-group.js

```diff
@@ -0,0 +1,46 @@
+import contentfulClient from '../plugins/contentful.js';
+
+export const state = () => ({
+  links: {}
+});
+
+export const mutations = {
+  setLinks(state, data) {
+    for (const d in data) {
```

Comment:
As the action triggers the commit for each group, change this to:
```js
  setLinks(state, data) {
    state.links[data.identifier] = data.links;
  }
```
(And see the accompanying change in the action, below.)